content/labs/intro-to-ada/chapters/records.rst

```diff
@@ -415,7 +414,7 @@ keep track of assets.
     #. Procedure :ada:`Add` adds an item to the assets.
 
         #. Since we want to keep track of the assets, the implementation must
-           accumulate the total amount of each item.
+           accumulate the total value of each item's inventory, the product of the item quanitity and its price.
```

Comment:
```suggestion
           accumulate the total value of each item's inventory, which is the 
           multiplication of the item quantity and its price.
```
Minor suggestion: I'd replace "product" by "multiplication" in this case. Also, there was a typo in "quantity."

Reply:
I agree that "product" could be confused with "item" in this domain. 

What do you think of "result of multiplying the item quantity and its price".

Reply:
This sounds good.